shaq_overflow/app/controllers/welcome_controller.rb

```diff
@@ -0,0 +1,5 @@
+class WelcomeController < ApplicationController
+  def index
+    @user = current_user
```

Comment:
You declared current_user as a helper_method in application controller. So it's available in views. You don't need an instance variable here - the view can just reference current_user.